src/components/Header/components/HeaderAction/index.tsx

```diff
@@ -11,24 +11,48 @@ import {
 import { LOGOUT } from '../../../../constants'
 
 export const HeaderAction = (): JSX.Element => {
-  const [isFullScreen] = useState(false)
+  const [widthWindow, setWidthWindow] = useState(window.screen.width)
+  const [isFullScreen, setIsFullScreen] = useState(!!document.fullscreenElement)
+
+  const handleScreen = () => {
+    if (document.fullscreenElement === null) {
+      setIsFullScreen(true)
+      document.documentElement.requestFullscreen()
+    } else if (document.fullscreenElement) {
+      setIsFullScreen(false)
+      document.exitFullscreen()
+    }
+  }
+
+  const handleWidth = useCallback(() => {
+    setWidthWindow(window.screen.width)
+  }, [])
+
+  useEffect(() => {
+    window.addEventListener('resize', handleWidth)
+    return () => window.removeEventListener('resize', handleWidth)
+  }, [])
+
+  console.log(widthWindow)
 
   return (
     <HeaderContainer>
       <UserInfo>
         some@email.com<Separator>:</Separator>sublogin
       </UserInfo>
       <LogoutButton>
-        {LOGOUT}
+        {widthWindow > 768 && `${LOGOUT}`}
         <Logout />
       </LogoutButton>
-      <ScreenButton>
-        {isFullScreen ? (
-          <DefaultScreen className="screen-icon" />
-        ) : (
-          <FullScreen className="screen-icon" />
-        )}
-      </ScreenButton>
+      {document.fullscreenEnabled && (
+        <ScreenButton onClick={handleScreen}>
+          {isFullScreen ? (
+            <DefaultScreen className="screen-icon" />
+          ) : (
+            <FullScreen className="screen-icon" />
+          )}
```

Comment:
Please format ternary operator. Follow this rule please. 'Ternaries should not be nested and generally be single line expressions.' Each expression from a new line.